DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateCourses/_CentreCourseCardDetails.cshtml

```diff
@@ -38,19 +38,19 @@
 
   <div class="nhsuk-summary-list__row">
     <dt class="nhsuk-summary-list__key">
-      Total delegates
+      In progress
     </dt>
-    <dd class="nhsuk-summary-list__value" name="delegate-count">
-      @Model.DelegateCount
+    <dd class="nhsuk-summary-list__value" name="in-progress-count">
+      @Model.InProgressCount
     </dd>
   </div>
 
   <div class="nhsuk-summary-list__row">
     <dt class="nhsuk-summary-list__key">
-      In progress
+      Completed
     </dt>
-    <dd class="nhsuk-summary-list__value" name="in-progress-count">
-      @Model.InProgressCount
+    <dd class="nhsuk-summary-list__value" name="completed-count">
+      @Model.CompletedCount

```

Comment:
Also, with the change from "Total delegates" to "Completed", the sorting options will no longer make sense here as it will still show "Total delegates" and I'm not sure the JS will still work.